katip/src/Katip/Monadic.hs

```diff
@@ -511,7 +511,7 @@ instance MonadIO m => KatipContext (NoLoggingT m) where
 
 -- | Convenience function for when you have to integrate with a third
 -- party API that takes a generic logging function as an argument.
-askLoggerIO :: (Applicative m, KatipContext m) => m (Severity -> LogStr -> IO ())
+askLoggerIO :: (Applicative m, KatipContext m, MonadIO m) => m (Severity -> LogStr -> IO ())
```

Comment:
I don't think this requires `MonadIO`, right? You return an `IO` function, but the constraint doesn't seem needed to do that.

It looks similar to https://hackage.haskell.org/package/log-base-0.12.0.1/docs/Log-Monad.html#v:getLoggerIO.